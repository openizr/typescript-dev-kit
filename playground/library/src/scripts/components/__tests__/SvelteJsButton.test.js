/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/svelte';
import SvelteTsButton from 'scripts/components/SvelteJsButton.svelte';

describe('svelte/SvelteTsButton', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('renders correctly - basic', () => {
    const { container } = render(SvelteTsButton);
    expect(container.firstChild).toMatchSnapshot();
  });
});
