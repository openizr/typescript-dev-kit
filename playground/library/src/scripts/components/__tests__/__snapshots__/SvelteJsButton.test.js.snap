// Vitest Snapshot v1

exports[`svelte/SvelteTsButton > renders correctly - basic 1`] = `
<div>
  <button
    class="s-IFZIjLgJOD5U"
  >
    Hello
  </button>
  <!--&lt;SvelteJsButton&gt;-->
</div>
`;
